eth_dummy_phy/examples/generic_switch/main/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+idf_component_register(SRCS "generic_switch.c"
+                       PRIV_REQUIRES esp_netif driver esp_eth)
```

Comment:
This is old notation - Use specific driver but ensure backward compatibility with older IDFs if required.